src/ai/backend/client/cli/admin/user.py

```diff
@@ -206,6 +208,15 @@ def list(ctx: CLIContext, status, group, filter_, order, offset, limit) -> None:
         '(e.g., --allowed-ip "127.0.0.1","127.0.0.2",...)'
     ),
 )
+@click.option(
+    "--enable-sudo-session",
+    is_flag=True,
+    default=False,
+    help=(
+        "Enable passwordless sudo for a user inside a compute session. "
+        "Note that this feature does not automatically install sudo for the session."
+    ),
+)
 @click.option("--description", type=str, default="", help="Description of the user.")
```

Comment:
```suggestion
@click.option("--description", type=str, default="", help="Description of the user.")
@click.option(
    "--enable-sudo-session",
    is_flag=True,
    default=False,
    help=(
        "Enable passwordless sudo for a user inside a compute session. "
        "Note that this feature does not automatically install sudo for the session."
    ),
)
```

It's not important, but I'd appreciate it if you could modify it to match the param order.